### PR TITLE
SW-4021 Move scheduled observations notification to the observations root page

### PR DESCRIPTION
**Pull request by karthikbtf**
- was previously only included in the list/map views
- move it up so it shows for the empty state as well

Terraware App 2023-08-08 10-04-58

### CODE REVIEW

File: src/components/Observations/ObservationsDataView.tsx [hunk 1]
```diff
@@ -48,31 +41,19 @@ export default function ObservationsDataView(props: ObservationsDataViewProps):
...
 
-  const observationsEvents = useMemo<ObservationEvent[]>(() => {
-    if (!upcomingObservations) {
-      return [];
-    }
-    const now = Date.now();
-    // return observations that haven't passed
-    return upcomingObservations.filter((observation) => now <= new Date(observation.endDate).getTime());
-  }, [upcomingObservations]);
-
```
Comment (karthikbtf) [inline src/components/Observations/ObservationsDataView.tsx:59]: moved to the parent page

File: src/components/Observations/ObservationsDataView.tsx [hunk 2]
```diff
@@ -48,31 +41,19 @@ export default function ObservationsDataView(props: ObservationsDataViewProps):
...
+      search={<Search {...searchProps} />}
+      list={<OrgObservationsListView observationsResults={observationsResults} />}
+      map={
+        selectedPlantingSiteId === -1 ? (
+          <AllPlantingSitesMapView />
+        ) : (
+          <ObservationMapView observationsResults={observationsResults} {...searchProps} />
+        )
+      }
+    />
```
Comment (karthikbtf) [inline src/components/Observations/ObservationsDataView.tsx:56]: removed the parent grid/events

File: src/components/PlantingSites/PlantingSiteView.tsx [hunk 3]
```diff
@@ -131,8 +131,10 @@ export default function PlantingSiteView(): JSX.Element {
           </Grid>
         )}
         {plantingSite?.boundary && !plantingSite.plantingZones && (
-          <Grid item xs={12}>
-            <SimplePlantingSite plantingSite={plantingSite} />
+          <Grid container flexGrow={1}>
+            <Grid item xs={12} display='flex'>
+              <SimplePlantingSite plantingSite={plantingSite} />
+            </Grid>
```
Comment (karthikbtf) [inline src/components/PlantingSites/PlantingSiteView.tsx:137]: oops, will remove

**Review by constanzauanini — APPROVED** [review]
Nice! Question, should the empty state say "No completed observations yet" for these cases maybe?

**Comment by karthikbtf** [comment]
Not sure why this deploy was not triggered.

**Pull request merged**